app/api/endpoints/plugins.py

```
@@ -0,0 +1,58 @@
+
+@router.delete("/{config_id}", status_code=status.HTTP_204_NO_CONTENT)
+async def delete_plugin_config(
+    config_id: str,
+    user_id: str = Depends(get_current_user_id),
+):
+    deleted = await PluginService.delete_plugin_config(config_id, user_id)
+    if not deleted:
+        raise HTTPException(status_code=404, detail="Plugin configuration not found")
+    return
+
+@router.get("/", response_model=List[PluginConfigurationOut])
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)